com.unity.netcode.adapter.utp/Runtime/NetworkMetricsPipelineStage.cs

```diff
@@ -0,0 +1,53 @@
+#if MULTIPLAYER_TOOLS
+using AOT;
+using Unity.Burst;
+using Unity.Collections.LowLevel.Unsafe;
+using Unity.Networking.Transport;
+using UnityEngine;
+
+namespace Unity.Netcode
+{
+    [BurstCompile]
+    public unsafe struct NetworkMetricsPipelineStage : INetworkPipelineStage
+    {
+        static TransportFunctionPointer<NetworkPipelineStage.ReceiveDelegate> ReceiveFunction = new TransportFunctionPointer<NetworkPipelineStage.ReceiveDelegate>(Receive);
+        static TransportFunctionPointer<NetworkPipelineStage.SendDelegate> SendFunction = new TransportFunctionPointer<NetworkPipelineStage.SendDelegate>(Send);
+        static TransportFunctionPointer<NetworkPipelineStage.InitializeConnectionDelegate> InitializeConnectionFunction = new TransportFunctionPointer<NetworkPipelineStage.InitializeConnectionDelegate>(InitializeConnection);
+
+        public NetworkPipelineStage StaticInitialize(byte* staticInstanceBuffer, int staticInstanceBufferLength, NetworkSettings settings)
+        {
+            return new NetworkPipelineStage(ReceiveFunction,
+                                            SendFunction,
+                                            InitializeConnectionFunction,
+                                            ReceiveCapacity: 0,
+                                            SendCapacity: 0,
+                                            HeaderCapacity: 0,
+                                            0);
+        }
+
+        public int StaticSize => 0;
+
+        [BurstCompile(DisableDirectCall = true)]
+        [MonoPInvokeCallback(typeof(NetworkPipelineStage.ReceiveDelegate))]
+        private static void Receive(ref NetworkPipelineContext networkPipelineContext, ref InboundRecvBuffer inboundReceiveBuffer, ref NetworkPipelineStage.Requests requests, int systemHeaderSize)
+        {
+            
+        }
+
+        [BurstCompile(DisableDirectCall = true)]
+        [MonoPInvokeCallback(typeof(NetworkPipelineStage.SendDelegate))]
+        private static int Send(ref NetworkPipelineContext networkPipelineContext, ref InboundSendBuffer inboundSendBuffer, ref NetworkPipelineStage.Requests requests, int systemHeaderSize)
+        {
+            return 0;
+        }
+
+        [BurstCompile(DisableDirectCall = true)]
+        [MonoPInvokeCallback(typeof(NetworkPipelineStage.InitializeConnectionDelegate))]
+        private static void InitializeConnection(byte* staticInstanceBuffer, int staticInstanceBufferLength,
+            byte* sendProcessBuffer, int sendProcessBufferLength, byte* receiveProcessBuffer, int receiveProcessBufferLength,
+            byte* sharedProcessBuffer, int sharedProcessBufferLength)
```

Comment:
Nit: could you wrap these parameters up according to our coding standard? For reference:

When wrapping params, do not leave any on line with function name
When wrapping, only indent one stop (do not line up with paren)

Reply:
Although not mentioned in our standard either way, I'd recommend one parameter per line as well, as it's much easier to read than a variable number of parameters on each line.